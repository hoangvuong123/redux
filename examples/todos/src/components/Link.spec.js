import React from 'react'
import renderer from 'react-test-renderer';
import Link from './Link'

describe('Link component', () => {
  it('render Link component', () => {
      const link = renderer.create(<Link />).toJSON()
      expect(link).toMatchSnapshot();
  })
})
