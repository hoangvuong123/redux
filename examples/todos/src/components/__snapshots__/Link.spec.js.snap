// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link component render Link component 1`] = `
<button
  disabled={undefined}
  onClick={undefined}
  style={
    Object {
      "marginLeft": "4px",
    }
  }
/>
`;
